File: APVN.LeadsPlatform.Client/apvn-leadsplatform-client/src/app/models/AssigLeadToDealerModel.ts
export class AssigLeadToDealerModel{
    constructor(){
        this.leadsId = 0;
        this.dealerId = 0;
        this.leadsActionId = null;
        this.dealerName = '';
        this.lstDealerId = [];
        this.lstDealerName = [];
    }
    leadsId: number;
    dealerId: number;
    leadsActionId: number;
    dealerName: string;
    dealerNote: string;
    leadActionPaymentStatus: number;
    lstDealerId: number[];
    lstDealerName: string[];
}

File: APVN.LeadsPlatform.Models/AssigLeadToDealerModel.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace APVN.LeadsPlatform.Models
{
    public class AssigLeadToDealerModel
    {
        public int LeadsId { get; set; }
        public int DealerId { get; set; }
        public int LeadsActionId { get; set; }
        public string DealerName { get; set; }
        public string DealerNote { get; set; }
        public int LeadActionPaymentStatus { get; set; }
        public List<int> LstDealerId { get; set; }
        public List<string> LstDealerName { get; set; }
    }
}
